katana_requirements.yaml

```diff
@@ -205,11 +205,11 @@ libarrow-python:
     conda: null
     pip: null
     apt: libarrow-python-dev
-  version: &arrow_version [ 4, 5 ]
+  version: &arrow_version [ 6, 7 ]
   labels:
     - apt
   version_overrides: &arrow_version_overrides
-    apt: "=4.0.1-1"
```

Comment:
Did you check that this is the exact version of the debian arrow packages? Not every version has a fast micro release like 4 did.

Reply:
Will check, thanks for the flag.

Reply:
Ok; `apt list` agrees that this is a valid version (and appears to be the latest for 6):
```
libarrow600/focal 6.0.1-1 amd64
```